core/src/main/java/com/oath/oak/OakUnsafeDirectBuffer.java

```diff
@@ -0,0 +1,59 @@
+package com.oath.oak;
+
+import java.nio.ByteBuffer;
+import java.util.function.Function;
+
+/**
+ * This interface allows high performance access to the underlying data of Oak.
+ * To achieve that, it sacrifices safety, so it should be used only if you know what you are doing.
+ * Misuse of this interface might result in corrupted data, a crash or a deadlock.
+ *
+ * Specifically, the developer should be concerned by two issues:
+ * 1. Concurrency: using this interface inside the context of serialize, compute, compare and transform is thread safe.
+ *    In other contexts (e.g., zero copy API), the developer should ensure that there is no concurrent access to this
```

Comment:
I wouldn't mention here "zero copy API", but you can speak about get/iterator output.

Reply:
fixed